docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -116,21 +164,25 @@ DESTINATION__SNOWFLAKE__CREDENTIALS__WAREHOUSE=""
 DESTINATION__SNOWFLAKE__CREDENTIALS__ROLE=""
 ```
 
-Ensure that these are defined in your environment.
+Ensure that these variables are defined in your environment.
+
+---
 
-## 4. Define a `DagsterDltResource`
+## Step 4: Define a DagsterDltResource
 
-The <PyObject module="dagster_embedded_elt.dlt" object="DagsterDltResource" /> provides a wrapper of a _dlt_ pipeline runner. This can be shared across _dlt_ pipelines, and is a required parameter to our _dlt_ asset definition.
+The <PyObject module="dagster_embedded_elt.dlt" object="DagsterDltResource" /> provides a wrapper of a dlt pipeline runner. This can be shared across dlt pipelines and is a required parameter to our dlt asset definition.
```

Comment:
What file should users be doing this in?

Reply:
This one I wasn't sure of, because it would be in a hypothetical `resource.py` file, or wherever the user typically defines their resources. In that sense, this "tutorial" isn't so much a tutorial as we don't prescribe the file structure. This could be changed, though.